labshare-client/src/components/InputForm.vue

```diff
@@ -1,12 +1,38 @@
 <i18n>
 {
-  "en": {
-    
-  },
-  "de": {
-    
-  }
-}
+    "en": {
+    "labName": "Name of laboratory",
+    "labWebsite": "Laboratory website",
+    "firstName": "First name",
+    "lastName": "Last name",
+    "phone": "Phone number",
+    "contactInfo": "Contact information",
```

Comment:
These should be in title case, which in this case means the nouns *should* be capitalised...